spring-boot-project/spring-boot-test-autoconfigure/src/main/java/org/springframework/boot/test/autoconfigure/web/reactive/WebFluxTest.java

```diff
@@ -43,7 +43,7 @@
  * <p>
  * Using this annotation will disable full auto-configuration and instead apply only
  * configuration relevant to WebFlux tests (i.e. {@code @Controller},
- * {@code @ControllerAdvice}, {@code @JsonComponent}, {@code Converter}, and
+ * {@code @ControllerAdvice}, {@code @JsonComponent}, {@code Converter}, {@code GenericConverter}, and
```

Comment:
I initially wrote that and I removed it as I felt it was a bit of a noise. So yeah if we scan a "converter" that includes the `GenericConverter` companion interface. Perhaps there is a better way to phrase that?

Reply:
@snicoll The reason why I added `{@code GenericConverter}` was that `{@code Converter}` is decorated with `@code` and as a code it didn't look to include `GenericConverter`. If it was "converter", it would be okay. I'm not sure how it can be improved ATM but maybe `{@code Converter}/{@code GenericConverter}`?

Reply:
Yeah I like that

Reply:
@snicoll Sure, will do.